providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -71,11 +71,15 @@
 )
 from airflow.providers.cncf.kubernetes.kube_config import KubeConfig
 from airflow.providers.cncf.kubernetes.kubernetes_helper_functions import annotations_to_key
-from airflow.stats import Stats
 from airflow.utils.log.logging_mixin import remove_escape_codes
 from airflow.utils.session import NEW_SESSION, provide_session
 from airflow.utils.state import TaskInstanceState
 
```

Comment:
I really hate to spring this on you so late into the project, but should these (here and elsewhere) be behind an AIRFLOW_V_3_2_PLUS check so we know when it's safe to remove them instead of using the try/except block?

Other than that i think it's ready to merge.

Reply:
Ah, my mistake, I'll definitely defer to Ash on that one.  Thanks!